vitest.config.ts

```diff
@@ -5,5 +5,6 @@ export default defineConfig({
 		coverage: {
 			reporter: ["lcovonly", "text"],
 		},
+		setupFiles: ["./vitest.setup"],
```

Comment:
I'm wondering if we cannot have the setup file inside of the test folder, just in order to minimize root files, wdyt? À-la AVA perhaps? https://github.com/prismicio/slice-simulator/blob/master/packages/core/test/__testutils__/setup.ts

Reply:
Works for me! Nesting the setup file in `__testutils__` might make it hard to find. What if we put it in the top level of `test/`, similar to `__fixtures__`, `__testutils__`, and `__snapshots__`?

```
test/__setup__.ts
```

Reply:
Works for me! Let go with it ☺️